content/kubermatic/master/startio/_index.en.md

```diff
@@ -0,0 +1,44 @@
++++
+title = "Start with KKP"
+weight = 1
+
++++
+
+![Kubermatic Kubernetes Platform logo](../img/KubermaticKubernetesPlatform-logo.jpg)
+
+## What is [start.kubermatic.io](https://start.kubermatic.io)?
+
+Application to simplify a way to bootstrap your Kubermatic Kubernetes Platform within a few minutes in an automated way operated with GitOps.
+
+This documentation provides you with the details of your journey.
+
+![High-level Flow](flow.png?width=700px&classes=shadow,border "High-level Flow")
+
+Check the [Concepts]({{< ref "./concepts/" >}}) page before starting to understand the used tools and prerequisites.
+
+Follow the steps in [Guides]({{< ref "./guides/" >}}) for a detailed explanation of all steps and understanding of generated content.
+
+Last but not least, there are a bunch of [Operations and Troubleshooting Guides]({{< ref "./cheat_sheets/" >}}) ready for you as well.
+
+## Disclaimer
+This project is in early adoption version at this stage. After collecting the initial feedback and improving the process,
+it will be fully open-sourced and published in the community.
```

Comment:
We should not mention the open sourcing yet, we may want to keep it closed depending of the outcome and the possibility to add a shop for buying EE.

Reply:
removed